<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsTiledSceneSourceSelectBase</class>
 <widget class="QDialog" name="QgsTiledSceneSourceSelectBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>383</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="srcGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Source Type</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout1">
      <item>
       <widget class="QRadioButton" name="mRadioSourceFile">
        <property name="text">
         <string>F&amp;ile</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mRadioSourceService">
        <property name="text">
         <string>Service</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="mStackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="fileGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Source</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Tiled scene dataset</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QgsFileWidget" name="mFileWidget" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="mConnectionsGroupBox">
         <property name="title">
          <string>Connections</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0" colspan="7">
           <widget class="QComboBox" name="cmbConnections"/>
          </item>
          <item row="1" column="5">
           <widget class="QToolButton" name="btnLoad">
            <property name="toolTip">
             <string>Load connections from file</string>
            </property>
            <property name="text">
             <string>Load</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>171</width>
              <height>30</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2">
           <widget class="QToolButton" name="btnEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Edit selected service connection</string>
            </property>
            <property name="text">
             <string>Edit</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QToolButton" name="btnDelete">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Remove connection to selected service</string>
            </property>
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QToolButton" name="btnNew">
            <property name="toolTip">
             <string>Create a new service connection</string>
            </property>
            <property name="text">
             <string>&amp;New</string>
            </property>
           </widget>
          </item>
          <item row="1" column="6">
           <widget class="QToolButton" name="btnSave">
            <property name="toolTip">
             <string>Save connections to file</string>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::NoButton</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbConnections</tabstop>
  <tabstop>btnNew</tabstop>
  <tabstop>btnEdit</tabstop>
  <tabstop>btnDelete</tabstop>
  <tabstop>btnLoad</tabstop>
  <tabstop>btnSave</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
